lib/resources/content/babelrc.js

```diff
@@ -2,26 +2,23 @@
 module.exports = () => {
   return {
     "plugins": [
+      "syntax-flow",
       "transform-decorators-legacy",
-      "transform-class-properties"
+      "transform-flow-strip-types"
     ],
     "presets": [
       [
         "env", {
           "targets": process.env.BABEL_TARGET === 'node' ? {
             "node": process.env.IN_PROTRACTOR ? '6' : 'current'
           } : {
-            "browsers": [
-              "last 2 versions",
-              "not ie <= 11"
-            ],
-            "uglify": process.env.NODE_ENV === 'production',
```

Comment:
As we are using terser now, we don't need to ask preset-env to generate compatible code for uglify-js v3.